features/steps/shell.py

```diff
@@ -64,7 +64,17 @@ def when_i_run_command(
 ):
     command = process_template_vars(context, command)
     prefix = get_command_prefix_for_user_spec(user_spec)
-    full_cmd = prefix + shlex.split(command)
+    if "pro" in command.split():
+        command = command.replace(
+            "pro",
```

Comment:
this would also turn `echo proposed` into `echo python3 -m coverage run  --source=/usr/lib/python3/dist-packages/uaclient /usr/bin/proposed`

I think we only want to check if `command.split[0] == "pro"`, replace the first item, and then `" ".join` it back together before `shlex.split`ing it.